back-end/slothstagram/settings.py

```diff
@@ -76,6 +79,13 @@
 
 WSGI_APPLICATION = 'slothstagram.wsgi.application'
 
+# CORS
+# https://www.stackhawk.com/blog/django-cors-guide/#enabling-cors-in-django
+CORS_ALLOWED_ORIGINS = [
+    "https://storied-blini-778bda.netlify.app/",
+    "http://localhost:8080",
+    "http://127.0.0.1:9000"
```

Comment:
What port is Django running on?
Here we say localhost at port 8080 then localhost again as 9000